From b90cf4cb456aa9cd2f5ebbce575663fc7a0ece03 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Sun, 19 Aug 2018 21:41:20 +0300
Subject: [PATCH] version-hooks: add Linux

Linux works the same way as U-Boot in terms of versioning.
Add it to the version friends group.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/package/version-hooks/version-hooks.mk b/package/version-hooks/version-hooks.mk
index 4a8ab1b..22987a7 100644
--- a/package/version-hooks/version-hooks.mk
+++ b/package/version-hooks/version-hooks.mk
@@ -7,4 +7,5 @@
 endef
 
 UBOOT_POST_RSYNC_HOOKS += HL_VERSION_GEN
+LINUX_POST_RSYNC_HOOKS += HL_VERSION_GEN
 
